app/view_models/prior_authority/v1/application_summary.rb

```diff
@@ -92,6 +92,14 @@ def current_section(current_user)
       def can_edit?(caseworker)
         submission.state == 'in_progress' && submission.assignments.find_by(user: caseworker)
       end
+
+      def unassignable?
+        submission.state == 'in_progress'
+      end
+
+      def assignable?
+        submission.state.in?(%w[submitted provider_updated])
```

Comment:
think long term I would prefer to have these as named things in a module..

i.e.
```
module States
  SUBMITTED = 'submitted'
  PROVIDER_UPDATED = 'provider_updated'

  ASSIGNABLE = [SUBMITTED, PROVIDER_UPDATED]
end
```